gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/filter/Any.feature

```diff
@@ -0,0 +1,156 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassFilter @StepAny
+Feature: Step - any()
+
+  Scenario: g_V_valuesXageX_anyXgtX32XX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().values("age").any(P.gt(32))
+      """
+    When iterated to list
+    Then the result should be empty
+
+  Scenario: g_V_valuesXageX_order_byXdescX_fold_anyXeqX29XX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().values("age").order().by(desc).fold().any(P.eq(29))
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | l[d[35].i,d[32].i,d[29].i,d[27].i] |
+
+  Scenario: g_V_valuesXageX_order_byXdescX_fold_anyXgtX10XX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().values("age").order().by(desc).fold().any(P.gt(10))
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | l[d[35].i,d[32].i,d[29].i,d[27].i] |
+
+  Scenario: g_V_valuesXageX_order_byXdescX_fold_anyXgtX42XX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().values("age").order().by(desc).fold().any(P.gt(42))
+      """
+    When iterated to list
+    Then the result should be empty
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectXabc_cdeX_anyXeqXbcdXX
+    Given the empty graph
+    And using the parameter xx1 defined as "l[abc,cde]"
```

Comment:
any particular reason to use a parameter here? doesn't `g.inject(['abc','cde'])...` work ok?